permissions/Base.permission

```diff
@@ -132,11 +132,12 @@ dbus-user.broadcast com.nokia.time=com.nokia.time.*@/*
 # BEG sessionbus-org.maliit.server.resource
 dbus-user.talk org.maliit.server
 dbus-user.broadcast org.maliit.server=org.maliit.server.*@/*
+whitelist ${RUNUSER}/maliit-server/dbus-socket
```

Comment:
Started to wonder what happens if maliit ever gets restarted? Does sandboxed app see a stale socket? Would something like allowing read-only access to containing directory work better?

Reply:
Tested these on device, if we whitelist the file, the keyboard doesn't reappear when maliit-server is restarted but then if I whitelist the directory it does. However it didn't stop maliit-server from working as keyboard was there once I restarted the app, unlike I anticipated on previous comment.

Reply:
After taking a peek at firejail source and some testing:

    whitelist DIR
    read-only DIR

seems to be the combination with desired outcome: sandboxed application gets a "live" view to the real directory, but can't modify anything underneath the DIR -> have notes app running and vkbd open; restart maliit -> vkbd goes away, and after a while comes back up and works; but things like firejail --put=$(pidof jolla-notes) foo /run/user/100000/maliit-server/bar fail due to EROFS

I guess the same applies for similar situations like e.g. the pulse audio socket in this PR.